src/components/Editable/useMultiline.ts

```diff
@@ -11,6 +9,10 @@ const useMultiline = (contentRef: React.RefObject<HTMLElement>, simplePath: Simp
   const fontSize = useSelector(state => state.fontSize)
   const showSplitView = useSelector(state => state.showSplitView)
   const splitPosition = useSelector(state => state.splitPosition)
+  const cursor = useSelector(state => state.cursor)
```

Comment:
I was afraid that this would cause some out-of-sync issues or race conditions again, but after excessive testing, couldn't find anything. Moved it back to `useSelectorEffect`.